dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/context/ContextManager.java

```diff
@@ -528,7 +582,7 @@ public TimedDirContext createDirContext(String principal, byte[] credential) thr
         Hashtable<String, Object> environment = getEnvironment(URLTYPE_SINGLE, activeURL);
         environment.put(Context.SECURITY_PRINCIPAL, principal);
         environment.put(Context.SECURITY_CREDENTIALS, credential);
-
+        environment.put(Context.SECURITY_AUTHENTICATION, ConfigConstants.CONFIG_AUTHENTICATION_TYPE_SIMPLE);
```

Comment:
This method is now for checking the password of the user logging in, correct?

Reply:
Correct, this method is only called by `authenticateWithPassword` and I updated the javadoc as such.